src/masoniteorm/schema/grammars/SQLiteGrammar.py

```diff
@@ -70,172 +70,6 @@ class SQLiteGrammar(BaseGrammar):
 
     null_map = {False: "NOT NULL", True: "NULL"}
 
-    """
```

Comment:
Do all this code was removed in replacement by new platform structure? :o

Reply:
yes this is in the middle of a redesign still but the grammar classes will now no longer be responsible for compiling queries. This is now the job of the platform class

Reply:
I removed all the tests for alter tables for all drivers and I will be adding it all back using the new structure.